### DISCUSSION TITLE
do.normalise

### DISCUSSION THREAD

**Discussion by ximle** [ImmuneDynamics/Spectre#69] [Q&A]
Hello Thomas,
Thank you for this very useful package!!! 
I read in your article about the do.normalise function, but I cannot run it. I think the normalization of my values will help me for visualization of the data in the multi-plots and heat maps. Any suggestion for normalizing the values? 
Thank you
Ximena

**Comment by tomashhurst** [comment]
Hi @ximle , do you mean getting the values in each _column_ to have the same distribution (e.g. between 0 and 1)? If so, the function you want is `do.rescale` (which our reviewers pointed out is a more accurate description of what is actually being done). You can run ```? do.rescale``` in R to get a description of how the function works.
